src/lib.rs

```diff
@@ -65,7 +65,7 @@
 //! }
 //! ```
 
-#![cfg_attr(feature = "unstable", feature(core))]
+#![cfg_attr(feature = "unstable", feature(core_intrinsics))]
```

Comment:
Wow, for how long has this been broken?

Reply:
TL;DR: dunno.
The only relevant changelog entry I could find was "Declaration of lang items and intrinsics are now feature-gated by default." in 0.11.0 (2014-07-02), but that's probably about feature(core) not feature(core_intrinsics). I could not find anything about core_intrinsics on Rust bug tracker.